app/components/file/browse/display/file-list-display-file-type-filter.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from "@angular/core";
```

Comment:
You used a pipe to filter those? I think it would be better to do a server side filter as we can do file startwtih applications, startup,etc.
How does this pipe work if the first batch of file doesn't actually contains some of those categories